### PR TITLE
Create AboutUs Page

### LINKED ISSUE #41: Create About Us Page
Use headshots and team descriptions from https://docs.google.com/document/d/1EKO8viWPpm-Y9r8ayLN-dGKZsnVy5rC8/edit#

### PR DESCRIPTION
**Pull request by sarinnasung**
Create an About Us Page

### CODE REVIEW

File: src/components/AboutTeam.tsx [hunk 1]
```diff
@@ -0,0 +1,213 @@
...
+        <div className="grid">
+          <div className = "wrapper">
+            <ProfileCard
+              imgUrl = {ConnorChongImg}
+              title = {"Connor Chong"}
+              description={"Founder | CEO"}
+              info= {"#connor"}
+              id={"connor"}
+              infoadded={"my name is connor"}
+              />
```
Comment (crchong1) [inline src/components/AboutTeam.tsx]: Small thing but can you set up your auto-formatter?

File: src/components/AboutTeam.tsx [hunk 2]
```diff
@@ -0,0 +1,213 @@
...
+      <div className="profile-sizing profile-body">
+        <div className="grid">
+          <div className = "wrapper">
+            <ProfileCard
+              imgUrl = {ConnorChongImg}
+              title = {"Connor Chong"}
+              description={"Founder | CEO"}
+              info= {"#connor"}
+              id={"connor"}
+              infoadded={"my name is connor"}
```
Comment (crchong1) [inline src/components/AboutTeam.tsx]: Just curious what is infoadded? and can we make it camelCase

File: src/components/AboutTeam.tsx [hunk 3]
```diff
@@ -0,0 +1,213 @@
...
+              <ProfileCard
+              imgUrl = {johnBaekImg}
+              title = {"John Baek"}
+              description={"Fundraiser & Grant Writer"}
+              info = {"#john"}
+              id = {"john"}
+              infoadded={"my name is john baek"}
+              />
+              <ProfileCard
+              imgUrl = {sambaDiaImg}
```
Comment (crchong1) [inline src/components/AboutTeam.tsx]: Can you also organize the cards by section?
Here are the sections we want:
Leadership and Board
Connor, Steffen, Daniel, Ivy, Michael, Emma
Board: Jackson, Jessica

Engineering
Gagan, Nick, Jason, Sarinna, and Tirtha

Design
Ying and Trang

Product
Cora

Legal and Ops
Samba

Marketing
Sean
Carolyn

Comment (ssung01-pixel) [reply src/components/AboutTeam.tsx:104]: Where should I put Kevin Chu?

File: src/components/AboutTeam.tsx (hunk 2 above)
Comment (sarinnasung) [reply src/components/AboutTeam.tsx]: Changed to personDescription

File: src/components/AboutTeam.tsx [hunk 4]
```diff
@@ -0,0 +1,283 @@
...
+      <div className="grid">
+        <div className="wrapper">
+          <ProfileCard
+            imgUrl={gaganKangImg}
+            title="Gagan Kang"
+            description="Front-end Developer"
+            info="#gagan"
+            id="gagan"
+            personDescription={
+              "My name is Gagandeep Kang and I'm a junior studying Computer Science with a minor in Mathematics from Boston University. As someone with an interest in Full-Stack Development, I'm always looking for ways to use my technical skills to build software that will help those around me. At BU, I've had the opportunity to create meaningful projects through our Hack4Impact Chapter as well as the Global App Initiative, and I'm looking forward to having more of these opportunities at Keep.id! "
```
Comment (crchong1) [inline src/components/AboutTeam.tsx]: did we want to wrap these into multiple lines like you did above?

**Review by crchong1 — APPROVED** [review]
lgtm

**Pull request merged**